lib/easel_provider.dart

```diff
@@ -751,6 +707,33 @@ class EaselProvider extends ChangeNotifier {
       );
     });
   }
+
+  Future saveNftAsset({required EaselProvider provider, required PageController controller}) async {
+    if (_file!.existsSync()) {
```

Comment:
Handle negative case at the top. 
Add proper return type. 
If this operation fail the execution should not go further.